lightning_pose_app/ui/extract_frames.py

```diff
@@ -825,3 +903,221 @@ def _render_streamlit_fn(state: AppState):
+            justify-content: center;
+            align-items: center;
+            gap: 10px;
+            margin-top: 10px;
+        }
+        .st-expander {
+            background-color: white !important;
+            border-radius: 8px;
+            padding: 20px !important;
+        }
+        .centered-container {
+            display: flex;
+            justify-content: center;
+        }
+        .title {
+            font-size: 28px;
+            font-weight: bold;
+            margin-top: 20px;
+            text-align: center;
+        }
+        .explanation {
+            font-size: 18px;
+            margin: 10px 20px;
+            text-align: center;
+        }
+        .button-explanation {
+            font-size: 16px;
+            margin-left: 10px;
+            margin-top: 10px;
+            text-align: left;
+        }
+        </style>
+        """,
+        unsafe_allow_html=True
+    )
+
+    st.divider()
+
+    st.header("Check Labeled Frames")
+    st_expander = st.expander("Expand for instractions")
+    with st_expander:
+        st.markdown(
+            """
+            Ensure accurate annotations with these steps:
+
+            1. **Select Keypoints**: Use the multiselect box to choose keypoints to check.
+
+            2. **Check Labels**: Click "Check labels" to upload and review frames, \
+                using arrows to navigate.
+
+            3. **Correct Annotations**: Go back to Label Studio to correct any wrong annotations.
+
+            4. **Download Frames**: Click "Download Frames" to save all annotated frames as\
+                a ZIP file.
+            """
+        )
+
+    collected_data_csv = os.path.join(state.proj_dir[1:], COLLECTED_DATA_FILENAME)
+
+    if os.path.exists(collected_data_csv):
+        # Adding multiselect box for body parts
+        df = pd.read_csv(collected_data_csv, header=[0, 1, 2])
+        body_parts = list(set(df.columns.get_level_values(1).tolist()[1:]))
+
+        # Adding multiselect box for body parts
+        selected_body_parts = st.multiselect(
+            "Select body parts to check",
+            options=["All"] + body_parts,
+            default=["All"],
+        )
+
+        st.write("**Click Check labels to upload and review the labeled frames.**")
+
+        st_start_check_labels = st.button(
+            "Check labels",
+            key="show_annotated_frames",
+            disabled=(not os.path.exists(collected_data_csv) or state.run_script_check_labels)
+        )
+        st.caption("This button will be enabled once labeled frames are available for review.")
+        if st_start_check_labels:
+            state.st_check_status = "initialized"
+            st.text("Request submitted!")
+            state.selected_body_parts = selected_body_parts
+            state.run_script_check_labels = True
+
+        st_autorefresh(interval=2000, key="refresh_check_labels")
+
+        labeled_data_check_path = os.path.join(state.proj_dir[1:], 'labeled-data-check')
```

Comment:
ah yes, good reason to change "labeled-data-check" to a variable that is imported